story.js

```diff
@@ -59,6 +60,9 @@ export const listenHack = () => {
   let currentFieldMusic = null;
   let currentFieldMusicIndex = 0;
   window.document.addEventListener('keydown', async e => {
+
+    if ( gameManager.inputFocused() ) return;

```

Comment:
There are better ways to route this. It's not the fault of this PR but there should be only one window `keydown` listener.

Reply:
should this be changed in this PR or different ?
I think different, cause this one is already quite big and this thing as you told not very related to the PR.

Reply:
Ok, but then in that case this line should also be removed from this PR.

Reply:
Missed this comment.
Already introduced the proper event handling to this PR.